main.nf

```diff
@@ -61,42 +61,43 @@ workflow NFCORE_RNAVAR {
+    ch_fasta_raw = params.fasta ? Channel.fromPath(params.fasta).map { it -> [[id: it.baseName], it] }.collect() : Channel.empty()
 
     // Initialize file channels based on params, defined in the params.genomes[params.genome] scope
-    ch_dict_raw       = params.dict                    ? Channel.fromPath(params.dict).map{ it -> [ [id:it.baseName], it ] }.collect()         : Channel.empty()
-    ch_fai_raw        = params.fasta_fai               ? Channel.fromPath(params.fasta_fai).map{ it -> [ [id:it.baseName], it ] }.collect()    : Channel.empty()
-    ch_dbsnp_raw      = params.dbsnp                   ? Channel.fromPath(params.dbsnp).map { dbsnp -> [[id:dbsnp.baseName], dbsnp]}.collect() : Channel.value([])
-    ch_known_indels_raw     = params.known_indels      ? Channel.fromPath(params.known_indels)                                                 : Channel.empty()
-    ch_known_indels_tbi_raw = params.known_indels_tbi  ? Channel.fromPath(params.known_indels_tbi)                                             : Channel.empty()
-    ch_gff            = params.gff                     ? Channel.fromPath(params.gff).map{ gff -> [ [ id:gff.baseName ], gff ] }.collect()     : Channel.empty()
-    ch_gtf_raw        = params.gtf                     ? Channel.fromPath(params.gtf).map{ gtf -> [ [ id:gtf.baseName ], gtf ] }.collect()     : Channel.empty()
-    ch_star_index_raw = params.star_index              ? Channel.fromPath(params.star_index).map { index -> [[id:index.baseName], index]}      : Channel.value([[],[]])
-    ch_exon_bed_raw   = params.exon_bed                ? Channel.fromPath(params.exon_bed).map { it -> [[id:it.baseName], it]}                 : Channel.empty()
+    ch_dict_raw = params.dict ? Channel.fromPath(params.dict).map { it -> [[id: it.baseName], it] }.collect() : Channel.empty()
+    ch_fai_raw = params.fasta_fai ? Channel.fromPath(params.fasta_fai).map { it -> [[id: it.baseName], it] }.collect() : Channel.empty()
+    ch_dbsnp_raw = params.dbsnp ? Channel.fromPath(params.dbsnp).map { dbsnp -> [[id: dbsnp.baseName], dbsnp] }.collect() : Channel.value([])
+    ch_known_indels_raw = params.known_indels ? Channel.fromPath(params.known_indels) : Channel.empty()
+    ch_known_indels_tbi_raw = params.known_indels_tbi ? Channel.fromPath(params.known_indels_tbi) : Channel.empty()
+    ch_gff = params.gff ? Channel.fromPath(params.gff).map { gff -> [[id: gff.baseName], gff] }.collect() : Channel.empty()
+    ch_gtf_raw = params.gtf ? Channel.fromPath(params.gtf).map { gtf -> [[id: gtf.baseName], gtf] }.collect() : Channel.empty()
+    ch_star_index_raw = params.star_index ? Channel.fromPath(params.star_index).map { index -> [[id: index.baseName], index] } : Channel.value([[], []])
+    ch_exon_bed_raw = params.exon_bed ? Channel.fromPath(params.exon_bed).map { it -> [[id: it.baseName], it] } : Channel.empty()
```

Comment:
Why did you change the formatting here?

Reply:
language server :shake_fist:

Reply:
![](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExZzRvdjU3c3pkYXljZzB3bGVmMnlhcmR1NmN1cXNnYWN2bGhjNWJ1ayZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3oEjHLIKODQJeCtEic/giphy.gif)